tests_v2/test_cli_new_workspace_command_queue.py

```
@@ -59,15 +60,37 @@ def _run_cli(cli: str, args: list[str]) -> tuple[subprocess.CompletedProcess[str
     created_ws_id: str | None = None
+    created_layout_ws_id: str | None = None
 
-    try:
-        marker.unlink(missing_ok=True)
-    except OSError:
-        pass
+    for path in (marker, layout_left_marker, layout_right_marker):
+        try:
+            path.unlink(missing_ok=True)
+        except OSError:
+            pass
```

Ruff (0.15.12)

[warning] Use `contextlib.suppress(OSError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(OSError): ...`

(SIM105)